.env.example

```diff
@@ -58,6 +58,9 @@ S3_VERSION=
 # Only let admins generated oauth clients
 KBIN_ADMIN_ONLY_OAUTH_CLIENTS=false
 
+# Let only authenticated users access contents
+MBIN_PRIVATE_INSTANCE=false
```

Comment:
Oh I thought you had removed these in https://github.com/MbinOrg/mbin/pull/678/commits/504a5bf16a5dc3b5f7d6d90d063d66201dce5701 but missed that you didn't. I don't think it should be needed here or in `.env.example_docker` anymore, the removal of https://github.com/MbinOrg/mbin/pull/678/commits/504a5bf16a5dc3b5f7d6d90d063d66201dce5701#diff-c47b03734cd2b4337b345eeba955637ba790d51fd98979f6d366d4acbdb104e0L163 means nothing looks for these anymore

Reply:
true, I missed it too, I will remove them